<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDefineType</class>
 <widget class="QDialog" name="UserDefineType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="MyWidgetRegister" name="widgetInner" native="true">
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>请输入自定义的类型名</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>请输入对此自定义类型的描述</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sure">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_close">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>×</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyWidgetRegister</class>
   <extends>QWidget</extends>
   <header location="global">mywidgetregister.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
